addons/website/static/src/scss/website.edit_mode.scss

```diff
@@ -282,6 +282,11 @@ body.editor_enable {
     .js_change_lang {
         pointer-events: none;
     }
+
+    // Disable "input group date" calendar icon
+    .input-group.date .input-group-text:has(.fa-calendar) {
```

Comment:
In website, we can't use `:has` in CSS because it's not well supported on Firefox yet but since it's for the "backend" (edit mode) and not a critical one anyway, I think it's ok.